modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractDartCodegen.java

```diff
@@ -164,7 +164,8 @@ public AbstractDartCodegen() {
                 "File",
                 "Client",
                 "Future",
-                "Response"
+                "Response",
+                "StreamedRequest"
```

Comment:
This should also go to the normal dart client as it is part of the `http` package and not used in the dio generator?

Reply:
This change is in AbstractDartCodegen.java so it affects all dart clients, including dartdio.

Reply:
Yea so it should also go to `DartClientCodegen`.

Reply:
I thought of this,  too. But then I saw [`Response`](https://pub.dev/documentation/http/latest/http/Response-class.html) which is also a `http` class (If I'm not wrong) so I thought I put it there. I have no problem to put it in the other set

Reply:
Yea, I added `Response` here because it also exists in Dio (a different class but same name). Same goes for `Client`.
I think it doesn't hurt here as the `http` package can basically be considered part of Dart.